tools/commit-message-lint

```diff
@@ -5,11 +5,9 @@
 
 # The rules can be found in /.gitlint
 
-if [[ "
-$(git remote -v)
-" =~ '
-'([^[:space:]]*)[[:space:]]*(https://github\.com/|ssh://git@github\.com/|git@github\.com:)zulip/zulip(\.git|/)?\ \(fetch\)'
-' ]]; then
+remote_url='([^[:space:]]*)[[:space:]]*(https://github\.com/|ssh://git@github\.com/|git@github\.com:)zulip/zulip(\.git|/)?\ \(fetch\)'
```

Comment:
Sorry, what I had in mind is just having `repository="zulip/zulip"` as the new variable, so that one can change just that simple line when porting this to a new repository.

Reply:
Okay, sure updating soon.